context/app/static/js/components/home-revision/FacetSearchMenu/FacetSearchMenu.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Skeleton from '@material-ui/lab/Skeleton';
+import List from '@material-ui/core/List';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+
+import FacetSearchEntityListItems from 'js/components/home-revision/FacetSearchEntityListItems';
+import { StyledPaper, StyledPopper, StyledAlert, StyledTypography, HeaderSkeleton } from './style';
+
+function FacetSearchMenu({ anchorRef, matches, labels, searchInputWidth, isLoading, menuIsOpen, setMenuIsOpen }) {
+  const matchesExist = Object.keys(matches).length > 0;
+
+  return (
+    <ClickAwayListener onClickAway={() => setMenuIsOpen(false)}>
+      <StyledPopper
+        id="facet-search-menu"
+        anchorEl={anchorRef.current}
+        open={menuIsOpen}
+        placement="bottom-start"
+        $searchInputWidth={searchInputWidth}
+      >
+        <StyledPaper>
+          {isLoading && (
+            <>
+              <StyledTypography variant="subtitle2">
+                <HeaderSkeleton />
+              </StyledTypography>
+              <StyledTypography variant="body2">
+                <Skeleton />
+              </StyledTypography>
+            </>
+          )}
+          {!isLoading && matchesExist && (
+            <List>
+              {Object.entries(matches).map(([k, v]) => (
+                <FacetSearchEntityListItems entityType={k} matches={v} labels={labels} />
+              ))}
+            </List>
+          )}
+          {!isLoading && !matchesExist && (
+            <StyledAlert severity="warning">No results found. Check your spelling or try a different term.</StyledAlert>
+          )}
```

Comment:
Consider just doing the `isloading` check once, and then a `matchesExist ? :` ternary.

Reply:
I avoided using a ternary because I thought you'd prefer to avoid them. I'll make the change.